cmd/pke/app/phases/runtime/kubernetes/kubelet_kernel_params.yaml.tmpl

```diff
@@ -1,3 +1,6 @@
 vm.overcommit_memory=1
+# vm.overcommit_ratio=100
+vm.oom_kill_allocating_task=1
```

Comment:
> If this is set to non-zero, the OOM killer simply kills the task that triggered the out-of-memory condition. This avoids the expensive tasklist scan.

Do we know that the tasklist scanning is too expensive? Or is it better to kill a random task that failed to allocate the last 1KB of memory instead of the one that caused the out of memory condition?

Reply:
Statement `one that caused the out of memory condition` does not stand. By default the Linux Kernel uses heuristics to determine which process gets killed or not. Setting `oom_score_adj` can lower the chance of being terminated, but that requires fine-tuning and it is based on PID. In general we have seen sshd or even Kuberentes CNI gets killed before the workload which caused the actual situation. The probability of a higher memory consumer process tries to allocate more memory is higher.

Reply:
this can be true (i'm not sure if this is a good solution), but then at least document it in the code